components/Dialogs/tests/unit/MDCAlertControllerCustomizationTests.m

```diff
@@ -96,16 +96,33 @@ - (void)testApplyingScrimColorToPresentationController {
   UIColor *scrimColor = [UIColor.orangeColor colorWithAlphaComponent:0.5];
   MDCDialogPresentationController *presentationController = alert.mdc_dialogPresentationController;
   if (presentationController == nil) {
-    return;  // don't fail the test if mdc_dialogPresentationController is empty
+    return;  // don't fail the test if mdc_dialogPresentationController is nil
   }
 
   // When
   presentationController.scrimColor = scrimColor;
 
   // Then
-  XCTAssertEqualObjects(alert.mdc_dialogPresentationController.scrimColor, scrimColor);
+  XCTAssertEqualObjects(presentationController.scrimColor, scrimColor);
 }
 
+- (void)testApplyingScrimColorToAlert {
+  // Given
+  UIColor *scrimColor = [UIColor.blueColor colorWithAlphaComponent:0.3];
+  MDCDialogPresentationController *presentationController = alert.mdc_dialogPresentationController;
+  if (presentationController == nil) {
+    return;  // don't fail the test if mdc_dialogPresentationController is nil
+  }
```

Comment:
_tip:_ Do we need these lines?

Reply:
which lines are you referring to? 
There are space lines between the Given/When/Then sections in all tests.

Reply:
I believe the return makes it so we can silently fail the test.

Reply:
Or silently succeed...

Any other suggestions? Can we mark this as resolved?